src/components/PopupWithForm.js

```diff
@@ -22,28 +24,34 @@ export default class PopupWithForm extends Popup {
   }
 
   open() {
-    this.handleLoading(false);
+    this.renderLoading(false);
     super.open();
   }
```

Comment:
* Please, delete method `open` from `PopupWithForm.js`. It's not needed there. You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case.